po/ar.po

```diff
@@ -1087,210 +1093,210 @@ msgstr "استخدام Cargo"
 
 #: src/SUMMARY.md src/error-handling/result.md src/bare-metal/aps/exceptions.md
 msgid "Exceptions"
-msgstr ""
+msgstr "استثناءات (Exceptions)"
 
 #: src/SUMMARY.md
 msgid "Useful Crates"
-msgstr ""
+msgstr "كريتات مفيدة ( Useful Crates)"
 
 #: src/SUMMARY.md src/bare-metal/useful-crates/zerocopy.md
 msgid "`zerocopy`"
-msgstr ""
+msgstr "`zerocopy`"
 
 #: src/SUMMARY.md src/bare-metal/useful-crates/aarch64-paging.md
 msgid "`aarch64-paging`"
-msgstr ""
+msgstr "`aarch64-paging`"
 
 #: src/SUMMARY.md src/bare-metal/useful-crates/buddy_system_allocator.md
 msgid "`buddy_system_allocator`"
-msgstr ""
+msgstr "`buddy_system_allocator`"
 
 #: src/SUMMARY.md src/bare-metal/useful-crates/tinyvec.md
 msgid "`tinyvec`"
-msgstr ""
+msgstr "`tinyvec`"
 
 #: src/SUMMARY.md src/bare-metal/useful-crates/spin.md
 msgid "`spin`"
-msgstr ""
+msgstr "`spin`"
 
 #: src/SUMMARY.md
 msgid "`vmbase`"
-msgstr ""
+msgstr "`vmbase`"
 
 #: src/SUMMARY.md
 msgid "RTC Driver"
-msgstr ""
+msgstr "برنامج تشغيل RTC (RTC Driver)"
 
 #: src/SUMMARY.md
 msgid "Concurrency: Morning"
-msgstr ""
+msgstr "التزامن: الصباح (Concurrency)"
 
 #: src/SUMMARY.md src/concurrency/threads.md
 msgid "Threads"
-msgstr ""
+msgstr "الخيوط البرمجية (Threads)"
 
 #: src/SUMMARY.md src/concurrency/threads/plain.md
 msgid "Plain Threads"
-msgstr ""
+msgstr "Plain Threads"
```

Comment:
I just skimmed this randomly and I think this is a nice comment — you should just translate it as "Threads" and then we should update the English source text. When you resync the Arabic translation, it will be easy to reconcile the translation.